config/samples/example_flags.json

```diff
@@ -84,6 +84,44 @@
           }, "binet", null
         ]
       }
+    },
+    "headerColor": {
+      "variants": {
+        "red": "#FF0000",
+        "blue": "#0000FF",
+        "green": "#00FF00",
+        "yellow": "#FFFF00"
+      },
+      "defaultVariant": "red",
+      "state": "ENABLED",
+      "targeting": {
+        "if": [
+          {
+            "$ref": "emailWithFaas"
+          },
+          {
+            "fractionalEvaluation": [
+              "email",
```

Comment:
I think the approach makes sense because it ensures that the value comes from the evaluation context. It does mean that things like `{ "var": "email" }` will not work as expected though. This should be fine as long as it's clearly documented.